### PR TITLE
Added PHPBench

### PR DESCRIPTION
**Pull request by dantleech**
Example PHPBench integration.

Can be useful as a base for checking for performance regressions.

### CODE REVIEW

**Comment by dbu** [comment]
cool. would it make sense to run that on travisci for just one of the builds?

**Comment by dantleech** [comment]
Added on two builds (one for doctrine_dbal and one for jackrabbit)

**Comment by dbu** [comment]
cool! 
argh, looks like travisci updated the phpunit version or something, we see lots of warnings in some builds.

the bench build seems to not work out: https://travis-ci.org/doctrine/phpcr-odm/jobs/135038598

**Comment by dantleech** [comment]
Thats a PHPBench error actually, it is trying to parse the PHPUnit
config but its in the wrong place. Updated, lets see what happens..

**Comment by dbu** [comment]
still no luck. i re-run styleci on master and merged the fixes. if you rebase, styleci should now be happy again.

**Comment by dantleech** [comment]
Trying again ...

**Comment by dbu** [comment]
looks good!

there are some build failures with phpunit warnings, but they have nothing to do with this change. what about this PR is still WIP? or can we merge it?

File: benchmarks/bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?php
+
+$transport = getenv('TRANSPORT') ?: null;
+
+
```
Comment (dantleech) [inline benchmarks/bootstrap.php]: remove extra line, add come comments.

**Comment by dantleech** [comment]
yeah, can be merged, hopefully it will be useful in other situations.

**Pull request merged**